### PR TITLE
fix: add Mohasin to the list

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/741"></a>

### CODE REVIEW

**Review by mohan-murali — APPROVED** [review]

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]
The file needs formatting...
If you have a local clone of the repo, it would be great if you can run prettier on this...
Or if you can edit the file with this version -
```
{
  "name": "Mohasin Haque",
  "bio": "Techie Teacher at Techie Tessie.",
  "avatar": "https://github.com/Mohasin-Haque.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/Mohasin-Haque",
      "icon": "github"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/mohasin_haque",
      "icon": "twitter"
    },
    {
      "name": "Follow me on Linkedin",
      "url": "https://www.linkedin.com/in/mohasin-haque-759786200/",
      "icon": "linkedin"
    },
    {
      "name": "Follow me on Facebook",
      "url": "https://www.facebook.com/ajad.ali.3766",
      "icon": "facebook"
    }
  ],
  "milestones": [
    {
      "title": "Started YouTube",
      "image": "https://github.com/Mohasin-Haque",
      "date": "August 2021",
      "icon": "youtube",
      "color": "red",
      "description": "Free programming videos every Friday",
      "url": "https://www.youtube.com/techietessie"
    }
  ]
}

```

**Review by Panquesito7 — DISMISSED** [review]
Please see the comments above.

**Comment by Mohasin-Haque** [comment]
I have formatted the file.. Now please correct me if I have to do anything more. Thanks for comments.

**Review by Vyvy-vi — APPROVED** [review]
looks good to me!

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
**Thanks for your contribution** 🚀

**Review by krishguptadev — COMMENTED** [review]
LGTM!

File: public/data/Mohasin-Haque.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "title": "Started YouTube",
+      "image": "https://github.com/Mohasin-Haque",
+      "date": "August 2021",
+      "icon": "youtube",
+      "color": "red",
+      "description": "Free programming videos every Friday",
+      "url": "https://www.youtube.com/techietessie"
+    }
+  ]
+}
```
Comment (Panquesito7) [inline public/data/Mohasin-Haque.json:38]: Missing an endline. I'll make a PR for it.
```suggestion
}

```

**Pull request merged**